src/components/Score.js

```diff
@@ -3,16 +3,40 @@ import { browserHistory } from 'react-router';
 
 import ScoreInput from './ScoreInput.js';
 
+const pushPayload = {
+  sessionId: null,
+  scopes: {}
+};
+
 const Score = React.createClass({
+  updatePushPayload(scope, score, comment) {
+    pushPayload.sessionId = this.props.params.sessionId;
+    pushPayload.scopes[scope] = pushPayload.scopes[scope] || {};
+    pushPayload.scopes[scope].score = score;
+    pushPayload.scopes[scope].comment = comment;
+  },
   submitScores() {
-    browserHistory.push('/d/' + this.props.params.sessionId);
+    // TODO: validation - pk2
+    fetch('http://localhost:3000/api/score', {
+      method: 'post',
```

Comment:
Yea, we can "agnostify" the env w/a local config in the future.

Re: save/score obj, I'll see what I can come up with in a quick refactor.